compiler error after update from repository

Just did a git pull and fresh build but got the following errors. Platform is CentOS 6 (latest/last) + dual Xeons + 32GB memory.

>`[ 98%] Building CXX object examples/cpp/11_native_cpp_kernels/CMakeFiles/examples_cpp_11_native_cpp_kernels.dir/main.cpp.o`
`.../occa/occa_1-clone/examples/cpp/11_native_cpp_kernels/main.cpp: In function int main(int, const char**):`
`.../occa/occa_1-clone/examples/cpp/11_native_cpp_kernels/main.cpp:41:4: `**error: converting false to pointer type for argument 2** `of occa::jsonKeyValue::jsonKeyValue(const string&, const char*) [-Werror=conversion-null]
   });`
`    ^
cc1plus: all warnings being treated as errors`
`make[2]: *** [examples/cpp/11_native_cpp_kernels/CMakeFiles/examples_cpp_11_native_cpp_kernels.dir/main.cpp.o] Error 1`
`make[2]: Target `examples/cpp/11_native_cpp_kernels/CMakeFiles/examples_cpp_11_native_cpp_kernels.dir/build' not remade because of errors.`
`make[1]: *** [examples/cpp/11_native_cpp_kernels/CMakeFiles/examples_cpp_11_native_cpp_kernels.dir/all] Error 2`

>`[ 98%] Built target cpp_example_12_addVectors_c`
`[ 98%] Building CXX object examples/cpp/12_native_c_kernels/CMakeFiles/examples_cpp_12_native_c_kernels.dir/main.cpp.o`
`.../occa/occa_1-clone/examples/cpp/12_native_c_kernels/main.cpp: In function int main(int, const char**):`
`.../occa/occa_1-clone/examples/cpp/12_native_c_kernels/main.cpp:44:4: `**error: converting false to pointer type for argument 2**` of occa::jsonKeyValue::jsonKeyValue(const string&, const char*) [-Werror=conversion-null]
   });`
`    ^
cc1plus: all warnings being treated as errors`
`make[2]: *** [examples/cpp/12_native_c_kernels/CMakeFiles/examples_cpp_12_native_c_kernels.dir/main.cpp.o] Error 1`
`make[2]: Target `examples/cpp/12_native_c_kernels/CMakeFiles/examples_cpp_12_native_c_kernels.dir/build' not remade because of errors.`

--- a/include/occa/types/json.hpp
+++ b/include/occa/types/json.hpp
@@ -591,6 +591,39 @@ namespace occa {
     json value;
 
     jsonKeyValue(const std::string &name_,
+                 const bool value_);
+
+    jsonKeyValue(const std::string &name_,
+                 const uint8_t value_);
+
+    jsonKeyValue(const std::string &name_,
+                 const int8_t value_);
+
+    jsonKeyValue(const std::string &name_,
+                 const uint16_t value_);
+
+    jsonKeyValue(const std::string &name_,
+                 const int16_t value_);
+
+    jsonKeyValue(const std::string &name_,
+                 const uint32_t value_);
+
+    jsonKeyValue(const std::string &name_,
+                 const int32_t value_);
+
+    jsonKeyValue(const std::string &name_,
+                 const uint64_t value_);
+
+    jsonKeyValue(const std::string &name_,
+                 const int64_t value_);
+
+    jsonKeyValue(const std::string &name_,
+                 const float value_);
+
+    jsonKeyValue(const std::string &name_,
+                 const double value_);
+
+    jsonKeyValue(const std::string &name_,
                  const primitive &value_);
 
     jsonKeyValue(const std::string &name_,

--- a/src/types/json.cpp
+++ b/src/types/json.cpp
@@ -668,6 +668,61 @@ namespace occa {
   }
 
   jsonKeyValue::jsonKeyValue(const std::string &name_,
+                             const bool value_) :
+    name(name_),
+    value(value_) {}
+
+  jsonKeyValue::jsonKeyValue(const std::string &name_,
+                             const uint8_t value_) :
+    name(name_),
+    value(value_) {}
+
+  jsonKeyValue::jsonKeyValue(const std::string &name_,
+                             const int8_t value_) :
+    name(name_),
+    value(value_) {}
+
+  jsonKeyValue::jsonKeyValue(const std::string &name_,
+                             const uint16_t value_) :
+    name(name_),
+    value(value_) {}
+
+  jsonKeyValue::jsonKeyValue(const std::string &name_,
+                             const int16_t value_) :
+    name(name_),
+    value(value_) {}
+
+  jsonKeyValue::jsonKeyValue(const std::string &name_,
+                             const uint32_t value_) :
+    name(name_),
+    value(value_) {}
+
+  jsonKeyValue::jsonKeyValue(const std::string &name_,
+                             const int32_t value_) :
+    name(name_),
+    value(value_) {}
+
+  jsonKeyValue::jsonKeyValue(const std::string &name_,
+                             const uint64_t value_) :
+    name(name_),
+    value(value_) {}
+
+  jsonKeyValue::jsonKeyValue(const std::string &name_,
+                             const int64_t value_) :
+    name(name_),
+    value(value_) {}
+
+  jsonKeyValue::jsonKeyValue(const std::string &name_,
+                             const float value_) :
+    name(name_),
+    value(value_) {}
+
+  jsonKeyValue::jsonKeyValue(const std::string &name_,
+                             const double value_) :
+    name(name_),
+    value(value_) {}
+
+  jsonKeyValue::jsonKeyValue(const std::string &name_,
                              const primitive &value_) :
     name(name_),
     value(value_) {}

